pkgs/by-name/ra/rainfrog/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  darwin,
+  fetchFromGitHub,
+  nix-update-script,
+  rustPlatform,
+  stdenv,
+}:
+let
+  version = "0.2.4";
+in
+rustPlatform.buildRustPackage {
+  inherit version;
+  pname = "rainfrog";
+
+  src = fetchFromGitHub {
+    owner = "achristmascarl";
+    repo = "rainfrog";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-3B56081ZiQPVFAheea2c7h2hQyruWI/q2crb4temVZc=";
+  };
+
+  cargoHash = "sha256-rO9tSgtO9q1ad0lzD8aINZhDupR5Q27ZPZPX/S7BM+I=";
+
+  buildInputs = lib.optionals stdenv.isDarwin (
+    with darwin.apple_sdk.frameworks;
+    [
+      Appkit
```

```suggestion
      AppKit
```
Build fails bc of this. `Appkit` isn't recognized. Trying to build right now using `AppKit`.

---

That builds successfully 👌 Just a typo. Just saw the conversation above

---

Thanks for the fix 😅